### PR TITLE
Update cloning-a-repository.md

### PR DESCRIPTION
**Pull request by yaakovab**
### Why: when I read that paragraph it became unclear to me because the sentence is not clear enough

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [v ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@yaakovab 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/repositories/creating-and-managing-repositories/cloning-a-repository.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ If you're unable to clone a repository, check that:
 
 - You can connect using HTTPS. For more information, see "[HTTPS cloning errors](/github/creating-cloning-and-archiving-repositories/https-cloning-errors)."
 - You have permission to access the repository you want to clone. For more information, see "[Error: Repository not found](/github/creating-cloning-and-archiving-repositories/error-repository-not-found)."
-- The default branch you want to clone still exists. For more information, see  have permission to access the repository you want to clone. For more information, see "[Error: Remote HEAD refers to nonexistent ref, unable to checkout](/github/creating-cloning-and-archiving-repositories/error-remote-head-refers-to-nonexistent-ref-unable-to-checkout)."
+- The default branch you want to clone still exists. For more information, see above paragraph: "have permission to access the repository you want to clone". For more information, see "[Error: Remote HEAD refers to nonexistent ref, unable to checkout](/github/creating-cloning-and-archiving-repositories/error-remote-head-refers-to-nonexistent-ref-unable-to-checkout)."
```
Comment (felicitymay) [inline content/repositories/creating-and-managing-repositories/cloning-a-repository.md:90]: I can see why you made this change, but I think this is an example of a copy and paste that went wrong. I think the correct fix is to delete this extra text and update the URL:

```suggestion
- The default branch you want to clone still exists. For more information, see "[Error: Remote HEAD refers to nonexistent ref, unable to checkout](/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors#error-remote-head-refers-to-nonexistent-ref-unable-to-checkout)."
```

**Review by felicitymay — COMMENTED** [review]
@yaakovab - many thanks for highlighting this error in the docs ✨

**Review by felicitymay — APPROVED** [review]

**Comment by felicitymay** [comment]
The preview looks good, [Cloning a repository](https://docs-preview-15691-9401b1.eastus.azurecontainer.io/en/repositories/creating-and-managing-repositories/cloning-a-repository#troubleshooting-cloning-errors), so this is ready to merge 🚀

**Comment by yaakovab** [comment]
so is it updated automatically? or should i do something?

‫בתאריך יום ג׳, 22 בפבר׳ 2022 ב-13:22 מאת ‪Felicity Chapman‬‏ <‪
***@***.***‬‏>:‬

**Comment by felicitymay** [comment]
My apologies for my unclear message @yaakovab 😞 

I've set this PR to merge automatically when the tests pass 🚀

**Comment by yaakovab** [comment]
Thanks for the quick response!!

**Comment by felicitymay** [comment]
Thanks for your help fixing this problem in the docs 💖

**Pull request merged**